cdap-docs/developers-manual/source/data-exploration/filesets.rst

```diff
@@ -71,12 +97,20 @@ Limitations
 -----------
 There are several limitations for fileset exploration:
 
-- All explorable files must be in Avro format.
-- Your version of Hive must include the AvroSerDe.
+- All explorable files must be in a format supported by your version of Hive. 
```

Comment:
I think here, we should only link to the Hive docs, not duplicate the content from there.